tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -346,14 +346,23 @@ internal static partial class ConfigurationKeys
         public const string StatsComputationInterval = "_DD_TRACE_STATS_COMPUTATION_INTERVAL";
 
         /// <summary>
-        /// Configuration key for setting the propagation style injection.
+        /// Configuration key for setting the header injection propagation style.
         /// </summary>
-        public const string PropagationStyleInject = "DD_PROPAGATION_STYLE_INJECT";
+        /// <seealso cref="Datadog.Trace.Propagators.ContextPropagationHeaderStyle"/>
+        /// <seealso cref="TracerSettings.PropagationStyleInject"/>
+        public const string PropagationStyleInject = "DD_TRACE_PROPAGATION_STYLE_INJECT";
 
         /// <summary>
-        /// Configuration key for setting the propagation style extraction.
+        /// Configuration key for setting the header extraction propagation style.
         /// </summary>
-        public const string PropagationStyleExtract = "DD_PROPAGATION_STYLE_EXTRACT";
+        /// <seealso cref="Datadog.Trace.Propagators.ContextPropagationHeaderStyle"/>
+        /// <seealso cref="TracerSettings.PropagationStyleExtract"/>
+        public const string PropagationStyleExtract = "DD_TRACE_PROPAGATION_STYLE_EXTRACT";
+
+        /// <summary>
+        /// Configuration key for setting the propagation style for both header injection and extraction.
```

Comment:
Maybe you could note here that values set by `PropagationStyleInject` and `PropagationStyleExtract` will override this value for the corresponding header operations?

Reply:
Done, thanks.